### PR TITLE
FIX: Correct return type for Member::currentUser()

### PR DESCRIPTION
**Pull request by kinglozzer**
See [this comment](https://github.com/silverstripe/silverstripe-framework/commit/995d07756d0b7aeef611d61fe3a31663b70e4c51#commitcomment-19754518).

### CODE REVIEW

**Comment by dhensby** [comment]
i think this needs to go against 3.4, then I can merge 3.4 -> 3.5.0 -> 3

**Comment by kinglozzer** [comment]
So, uh, changing the base branch didn’t quite go as expected... Lemme see if I can force push and fix it without opening a new PR 😝

**Comment by kinglozzer** [comment]
Ah, that’s better

**Comment by dhensby** [comment]
just rebasing and force pushing would work

File: security/Member.php [hunk 1]
```diff
@@ -837,9 +837,9 @@ public static function currentUser() {
 		$id = Member::currentUserID();
 
 		if($id) {
-			return DataObject::get_by_id('Member', $id);
+			$member = DataObject::get_by_id('Member', $id);
```
Comment (dhensby) [inline security/Member.php]: You could avoid this variable assignment and do `DataObject::get_by_id('Member', $id) ?: null;`

**Pull request merged**